services/ui-src/src/components/index.ts

```diff
@@ -30,7 +30,8 @@ export { DashboardList } from "./pages/Dashboard/DashboardProgramList";
 export { MobileDashboardList } from "./pages/Dashboard/DashboardProgramListMobile";
 // drawers
 export { Drawer } from "./drawers/Drawer";
-export { ReportDrawer } from "./drawers/ReportDrawer";
+export { ReportDrawer } from "./drawers/ReportDrawer/ReportDrawer";
+export { ReportDrawerDetails } from "./drawers/ReportDrawer/ReportDrawerDetails";
```

Comment:
Nit / Opinion: Because there's only one other kind of thing in drawer I think we can keep these un-nested in there. We don't do a ton of nesting in the component directories right now and if we start doing stuff like this it's a big pattern change. Not a strongly held opinion on how we should organize things though, so I'm cool with whatever.